pages/1.13/metrics/index.md

```diff
@@ -10,37 +10,61 @@ enterprise: false
 
 <!-- The source repo for this topic is https://github.com/dcos/dcos-docs-site -->
 
-Metrics in DC/OS, version 1.12 or newer, use [Telegraf](/1.13/overview/architecture/components/#telegraf) to collect and process data. Telegraf provides metrics from DC/OS cluster hosts, containers running on those hosts, and from applications running on DC/OS using the `statsd` process. Telegraf is natively integrated with DC/OS. By default, it exposes metrics in Prometheus format from `port 61091` on each node, and in JSON format through the DC/OS [Metrics API](/1.13/metrics/metrics-api/).
+# Overview
+DC/OS includes a comprehensive metrics service, providing metrics from DC/OS cluster hosts, containers, and applications running on those hosts. It collects, tags, and transmits metrics from every node, container, and application in your DC/OS cluster. Metrics are available via the DC/OS metrics API, allowing for easy integration with a wide range of monitoring solutions.
 
-## Overview
+# Types of Metrics
 DC/OS collects four types of metrics as follows:
 
 * **System:** Metrics about each node in the DC/OS cluster.
 * **Component:** Metrics about the components which make up DC/OS.
 * **Container:** Metrics about `cgroup` allocations from tasks running in the DC/OS [Universal Container Runtime](/1.13/deploying-services/containerizers/ucr/) or [Docker Engine](/1.13/deploying-services/containerizers/docker-containerizer/) runtime.
 * **Application:** Metrics emitted from any application running on the Universal Container Runtime.
 
-Telegraf is included in the DC/OS distribution and runs on every host in the cluster. Because Telegraf provides a plugin-driven architecture, custom DC/OS plugins provide metrics on the performance of DC/OS workloads and DC/OS itself.
+# Operations on Metrics 
+You can perform the following operations on Metrics:
 
-Telegraf collects application and custom metrics through the `dcos_statsd` plugin. A dedicated StatsD server is started for each new task. Any metrics received by the StatsD server are tagged with the task name and its service name. The address of the server is provided by environment variables (`STATSD_UDP_HOST` and `STATSD_UDP_PORT`). Note that when a task finishes, any metrics it has emitted that haven't yet been gathered by Telegraf will be discarded. The metrics collected by `dcos_statsd` are gathered every 30 seconds. To ensure a task's metrics are gathered, the task must run for at least 30 seconds.
+1. **Adding Metrics:** You can add custom metrics to the DC/OS metrics service from your applications. DC/OS metrics listens for `StatsD` metrics from every application running with the Mesos containerizer. A `StatsD server` is exposed for each container, which allows you to tag all metrics by origin. Its address is available to the application by injecting the standard environment variables `STATSD_UDP_HOST` and `STATSD_UDP_PORT`.
```

Comment:
I think we should use "StatsD" without monospacing it, as it's used in StatsD docs, e.g. https://github.com/statsd/statsd/blob/master/docs/backend.md. StatsD isn't a command or string literal, so I don't think it's helpful to wrap it in backticks.

Reply:
@gracedo @philipnrmn We support collecting workload metrics in Prometheus format, right? Should we add something here about that?

Reply:
We should! Let's draft something here for that.

Reply:
```suggestion
1. **Adding Metrics:** You can add custom metrics to the DC/OS metrics service from your applications. DC/OS metrics listens for `StatsD` metrics from every application running with the Mesos containerizer. A `StatsD server` is exposed for each container, which allows you to tag all metrics by origin. Its address is available to the application by injecting the standard environment variables `STATSD_UDP_HOST` and `STATSD_UDP_PORT`.

Alternatively, you can serve metrics in prometheus format by exposing an endpoint which serves metrics. Notify the metrics service that your task is serving prometheus metrics by adding a label to the port which is serving metrics. Any port with the label `DCOS_METRICS_FORMAT=prometheus` will be scraped at the rate of once per minute for the lifetime of the task. 

A task may expose multiple ports which serve prometheus metrics. Metrics collected in this manner will be tagged with the originating task's name and its framework name. 
```

Reply:
Here's a snippet of a task which exposes Prometheus format metrics:
```json
        "discovery": {
          "name": "task-with-metrics",
          "ports": {
            "ports": [
              { "number": 12345,
                "protocol": "tcp",
                "labels": {
                  "labels": [ {"key": "DCOS_METRICS_FORMAT","value": "prometheus"} ]
                }
              }
            ]
          }
        },
```